docs/usage/ssr-hooks.md

```diff
@@ -0,0 +1,96 @@
+---
+name: SSR Hooks
+menu: Usage
+route: '/usage/ssr-hooks'
```

Comment:
I think the route and the file should become just `/usage/ssr` (I haven't done it as a suggestion because I can't rename the file to match).

Reply:
If you do this, don't forger to update any links to this page (I saw at least one in `api-reference`)